src/models/Settings.js

```diff
@@ -52,6 +52,7 @@ function (Okta, Errors, BrowserFeatures) {
       'features.rememberMe': ['boolean', true, true],
       'features.rememberDevice': ['boolean', true, true],
       'features.rememberDeviceAlways': ['boolean', true, false],
+      'features.autoPush': ['boolean', true, false],
```

Comment:
this.model.get('autoPush') is accessing value in model `Factor` instead of model `Settings`, For now there's no default value specified for Factor.autoPush, so it would remain undefined if autoPush checkbox is not initialized. I can add one though.